tests/src/gretel_client/unit/transformers/test_data_pipeline.py

```diff
@@ -660,3 +660,60 @@ def test_record_fpe_base62():
     assert check == '5931468769662449'
     check = rf.transform_record(xf_payload)
     assert check == rec
+
+
+def test_record_fpe_mask():
+    rec = {'latitude': -70.783, 'longitude': -112.221, 'credit_card': '4123 5678 9123 4567', 'the_dude': 100000000,
+           'the_hotness': "convertme", "the_sci_notation": 1.23E-7}
+    cc_xf = [FormatConfig(pattern=r'\s+', replacement=''),
+             SecureFpeConfig(
+                 secret="2B7E151628AED2A6ABF7158809CF4F3CEF4359D8D580AA4F7F036D6F04FC6A94", radix=10,
+                 mask='*XXXXXXXXXXXXXXX', mask_fpe_char='X')]
```

Comment:
Is the leading `*` supported? I thought explicit masks were used

Reply:
It only checks for the character to be masked. Synonymous with the previous comment.